tests/check_framework/test_urls.py

```diff
@@ -4,12 +4,12 @@
 
 
 class CheckUrlsTest(SimpleTestCase):
-    @override_settings(ROOT_URLCONF='check_framework.urls_no_warnings')
+    @override_settings(ROOT_URLCONF='check_framework.urlconfs.no_warnings')
     def test_include_no_warnings(self):
         result = check_url_config(None)
         self.assertEqual(result, [])
 
-    @override_settings(ROOT_URLCONF='check_framework.urls_include')
+    @override_settings(ROOT_URLCONF='check_framework.urlconfs.include')
```

Comment:
I think the name could be a bit more specific like `include_with_dollar` to prevent clashes with possible future checks that involve include, slash, or name.

Maybe a `views.py` for the dummy view so we don't need to define it in every urls file would be an improvement as well (or even something like `lambda x: x` in place of an actual view might work).